Formula/kubeless.rb

```diff
@@ -0,0 +1,25 @@
+class Kubeless < Formula
+  desc "Kubernetes Native Serverless Framework"
+  homepage "https://github.com/kubeless/kubeless"
+  url "https://github.com/kubeless/kubeless/releases/download/v0.2.3/kubeless_darwin-amd64.zip"
+  version "0.2.3"
+  sha256 "0d8c19d8d08858c1508dfa892b11ee37f128c70563c2e7569c66fbde847844bf"
+  bottle :unneeded
+
+  def install
+    bin.install "kubeless_darwin-amd64/kubeless"
+  end
+
+  def caveats; <<-EOS.undent
+    To use kubeless you need a running Kubernetes cluster
+    Try `kubectl version` to make sure you are ready
+    Then launch the kubeless controller with:
+    `curl -sL https://github.com/kubeless/kubeless/releases/download/v#{version}/kubeless-v#{version}.yaml | kubectl create -f -`
+    EOS
+  end
+
+  test do
+    output = shell_output("#{bin}/kubeless version")
+    assert_match "Kubeless version: v#{version}", output
```

Comment:
Note that we cannot test much else since we need a Kubernetes environment configured locally in the machine

Reply:
This needs a real test of some kind. See faas-cli for how to mock a server.